compiler/damlc/lib/DA/Cli/Damlc/Test.hs

```diff
@@ -59,15 +60,22 @@ testRun h inFiles lfVersion color mbJUnitOutput  = do
     deps <- runActionSync h $ mapM getDependencies inFiles
     let files = nubOrd $ concat $ inFiles : catMaybes deps
 
-    results <- runActionSync h $
+    results <- runActionSync h $ do
+        dalfs <- Shake.forP files $
+          \file -> dalfForScenario file
+        let templates = [t | m <- dalfs , t <- NM.toList $ LF.moduleTemplates m]
```

Comment:
Maybe just pass down the dalfs to `printTestCoverage` and move this logic there?

Reply:
yes, why not.